build.psm1

```diff
@@ -681,13 +681,13 @@ function Publish-PSTestTools {
 
     Find-Dotnet
 
-    $tools = @("$PSScriptRoot/test/tools/EchoArgs", "echoargs"), @("$PSScriptRoot/test/tools/CreateChildProcess", "createchildprocess")
+    $tools = ,@("${PSScriptRoot}/test/tools/TestExe", "testexe")
```

Comment:
TestExe vs testexe - We need to be consistent and cautious with the case of letters.
TestExe may be a bad name on Linux. Maybe `BinaryTestRunner` or something like?